README.md

```diff
@@ -4,6 +4,48 @@ This project uses the
 [Trusted Web Activities](https://developers.google.com/web/updates/2017/10/using-twa) technology
 to wrap [SVGOMG](https://jakearchibald.github.io/svgomg/) in an Android Application.
 
+## Running the Demo
+
+1. Clone the project
+``
+git clone https://github.com/GoogleChromeLabs/svgomg-twa.git
+``
+
+2. Import the Project into Android Studio, using File > New > Import Project, and select the folder
+to which the project was cloned.
+
+3. Run the Project (Ctrl+R)
+
+### Enabling Debug
+
+TWAs require [Digital AssetLinks](https://developers.google.com/digital-asset-links/) to be setup
+on both the application and on the website, in order to enable the validation that allows Chrome to
+open the page in full-screen.
+
+For security reasons, the signing key compatible with the setup on
+https://svgomg.firebaseapp.com/ is not committed with the sample code.
+
+It is possible to setup Chrome to skip validation on device to enable testing.
+
+Here are the 2 steps required to achieve this:
+
+1. Enable Chrome to accept command-line parameters:
+
+On the Android Device, go to the Chrome version being used to test the TWA and navigate to
+`chrome://flags`. Search for a setting called `Enable commmand line on non-rooted devices` and
+change it to `Enabled`. Restarting the browser may be required.
+
+2. Create an Android file with the command-line parameters that allow skipping the TWA validation.
+
+Add a file at `/data/local/tmp/chrome-command-line`, with the content
+`_ --disable-digital-asset-link-verification-for-url="https://svgomg.firebaseapp.com"`.
```

Comment:
It's also worth pointing out that if they modify the file themselves, it may not work if there's a newline at the end of it (so we recommend using the script).